### PR TITLE
Loads the correct spotlight article

### PR DESCRIPTION
**Pull request by b-m-f**
Depends on https://github.com/canonical-web-and-design/canonicalwebteam.blog/pull/30
## Done

- Loads the correct article for the spotlight column

## QA

- `./run` and make sure that the spotlight article is different from the latest news articles

### CODE REVIEW

File: static/js/latest-news.js [hunk 1]
```diff
@@ -100,9 +100,9 @@ const createReqListenerForDomContainer = (
   }
 
   try {
-    const latestPinned = latest[0];
+    const latestPinned = data.latest_pinned_articles[0][0];
```
Comment (bartaz) [inline static/js/latest-news.js]: Is this array in array or a typo?

Comment (b-m-f) [reply static/js/latest-news.js]: Its actually nested like this. 
You can see the response here https://jp-ubuntu-com-canonical-web-and-design-pr-132.run.demo.haus/blog/latest-news :)

That is the reason I have wrapped this in a try/catch

Comment (bartaz) [reply static/js/latest-news.js]: OK. Bit weird and confusing structure of data, but I guess it works.

**Comment by bartaz** [comment]
I guess there is currently no spotlight article for QA?

**Pull request merged**